### DISCUSSION TITLE
Adjust text color

### DISCUSSION THREAD

**Discussion by jcannon23** [punxaphil/custom-sonos-card#23] [Q&A] [answered]
Hi - 

Thank you for developing this awesome custom card! I was wondering how I can update the text color on the card using the theme settings provided? Thanks!

**Comment by punxaphil** [comment, marked as answer]
🙂
https://www.home-assistant.io/integrations/frontend/

**Reply by jcannon23** [reply]
Would it be possible to provide an example of what the configure should look like? I tried the below and it doesn't work.

 

>  Sonos-card-test:
>     sonos-background-color: '#4C4C4D'
>     sonos-player-section-background: '#ffffffe6'
>     sonos-color: '#273D95'
>     sonos-artist-album-text-color: '#F34907'
>     sonos-accent-color: '#7EF307'
>     sonos-title-color: '#F1F1F1'
>     sonos-border-radius: '0.25rem'
>     sonos-border-width: '0.125rem'

**Reply by punxaphil** [reply]
Please read the link. You need to define the theme vars  in configuration.yaml file 🙂

**Comment by pimmen85** [comment]
me who aint no guru in yaml and homeassistant in general dont understand how to use the "settings" and create a theme at all, but would love to change a few things with the sonos player.
